docs/includes/pairlists.md

```diff
@@ -207,6 +208,8 @@ In "append" mode, the retrieved pairlist is added to the original pairlist. All
 
 The `pairlist_url` option specifies the URL of the remote server where the pairlist is located, or the path to a local file (if file:/// is prepended). This allows the user to use either a remote server or a local file as the source for the pairlist.
 
+The `save_to_file` option, when provided with a valid filename, saves the processed pairlist to that file in JSON format. This option is optional, and by default, the pairlist is not saved.
```

Comment:
i think you MIGHT wanna extend / reword this section ...

Starting with this (i red the docs change first) i understood:
"i give 3 bots on the same server the same filename, and they'll identify "do i need to reload this file or can i use it as is" - and then auto-load from here, OR fall back to a network request - refreshing the file.

That's however not really what's happening - the actual assumption is that 
* bot A fetches and saves to file
* bots B and C are configured with this file as "pairlist_url" ... 

Now they both use different syntax (`file:///somefile.json` vs. `./somefile.json`) - which i think will need some more explanation (or maybe a config sample for the 2 bot setups in a `??? Note` block - which defaults to collapsed)  to be actually clear.